asm/macros/event.inc

```diff
@@ -469,14 +469,14 @@
 	.endm
 
 	@ Attempts to add quantity of item index to the player's Bag. If the player has enough room, the item will be added and VAR_RESULT will be set to TRUE; otherwise, VAR_RESULT is set to FALSE.
-	.macro giveitem index:req, quantity:req
+	.macro additem index:req, quantity=1
```

Comment:
This name change is a very intrusive change for people like Diego, as any existing `giveitem` calls will still compile, but have different behavior.  The name change should still happen, since the names are better, but it will cause some people to tear their hair out if you don't communicate this.

Reply:
Addressed in Discord

Reply:
Updating macros per discussion, dont merge until resolved

Reply:
Macros updated. Should just be waiting on making sure people know about the change to avoid silent conflicts